test/deployer_tron.js

```diff
@@ -0,0 +1,19 @@
+const deployer = require("../tron/deployerImplementation.js");
+
+require("chai")
+    .use(require("chai-as-promised"))
+    .should()
+
+
+describe('Deployer', async () => {
+
+    it("test WBTC deployer script on Shasta network", async () => {
+        await deployer.deploy("tron/deployerInputTestrpc.json", 1000000000,  10000000,'https://api.shasta.trongrid.io', true, "WBTC");
```

Comment:
let's add a configuration value to the deployer that's similar to the energy used parameter:

```          consume_user_resource_percent: 30,```

is the tronbox attribute

Reply:
you can find more here: https://developers.tron.network/docs/tron-box-contract-deployment